samples/buildstrategy/buildah/buildstrategy_buildah_cr.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: build.dev/v1alpha1
+kind: BuildStrategy
+metadata:
+  name: buildah
+spec:
+  buildSteps:
+    - name: step-buildah-bud
+      image: $(build.builderImage)
```

Comment:
We aren't going to support setting the buildah image from Build CR, because we don't want it to overwritten.